src/server/rest.coffee

```diff
@@ -142,14 +155,35 @@ getDocument = (req, res, client) ->
       else
         sendError res, error
 
-# GET returns the documents historical snapshots. The type is sent as headers.
+# GET returns the document snapshots.
 getDocumentSnapshots = (req, res, client) ->
-  client.getSnapshots req.params.name, (error, snapshots) ->
-    if snapshots && snapshots[0]?
-      res.setHeader 'X-OT-Type', snapshots[0].type
-      sendJSON res, snapshots
+  client.getSnapshots req.params.name, (error, docs) ->
+    if docs && docs.length > 0
+      res.setHeader 'X-OT-Type', docs[0].type
+      if req.method == "HEAD"
+        send200 res, ""
+      else
+        sendJSON res, docs
     else
-      sendError res, error
+      if req.method == "HEAD"
+        sendError res, error, true
+      else
+        sendError res, error
+
+# GET returns the document versions.
+getDocumentVersions = (req, res, client) ->
+  client.getVersions req.params.name, req.params.every, (error, docs) ->
+    if docs && docs.length > 0
+      res.setHeader 'X-OT-Type', docs[0].type.name
+      if req.method == "HEAD"
+        send200 res, ""
+      else
+        sendJSON res, docs
+    else
+      if req.method == "HEAD"
+        sendError res, error, true
+      else
+        sendError res, error
```

Comment:
Is support for `HEAD` included for a specific use case or just to stick with the established pattern in this file?

Reply:
I was just following the pattern. It probably doesn't need to support `HEAD` requests.

Reply:
I'm all for following the existing patterns 👍